site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,384 @@
+# GEP-1426: HTTPRoute Mesh Binding
+
+* Issue: [#1294](https://github.com/kubernetes-sigs/gateway-api/issues/1294)
+* Status: Implementable
+
+## Overview
+
+Similar to how `HTTPRoutes` bind to `Gateways` and mange North/South traffic flows in Gateway API’s ingress use-case, it would be natural to adopt a similar model for traffic routing concerns in service mesh deployments. The purpose of this GEP is to add a mechanism to the Gateway API spec for the purpose of associating `HTTPRoutes` to a service mesh and offering a model for service owners to manage traffic splitting configurations.
```

Comment:
```suggestion
Similar to how `HTTPRoutes` bind to `Gateways` and mange North/South traffic flows in Gateway API’s ingress use-case, it would be natural to adopt a similar model for traffic routing concerns in service mesh deployments. The purpose of this GEP is to add a mechanism to the Gateway API spec for the purpose of associating `HTTPRoutes` to a service mesh and offering a model for service owners to manage traffic splitting configurations.

This GEP is intended to establish an implementable baseline for supporting basic service mesh traffic routing functionality through the Gateway API spec.
```
@robscott thinking about adding this to help clarify your question about when GAMMA might be considered "implementable" - my hope is that advancing this GEP would be enough to get started, even as we continue to work on increasing the scope of supported functionality.

Thoughts @howardjohn @keithmattix? Is it too early to add this, or would adding a caveat about enforcing AuthZ through implementation-specific configuration until supported in Gateway API be sufficient?